config/config.exs

```diff
@@ -46,7 +46,7 @@ oban_crontab = [
   {"0 * * * *", Glific.Jobs.MinuteWorker, args: %{job: :stats}},
   {"1 * * * *", Glific.Jobs.MinuteWorker, args: %{job: :hourly_tasks}},
   {"2 * * * *", Glific.Jobs.MinuteWorker, args: %{job: :delete_tasks}},
-  {"3 0 * * *", Glific.Jobs.MinuteWorker, args: %{job: :daily_tasks}},
+  {"58 23 * * *", Glific.Jobs.MinuteWorker, args: %{job: :daily_tasks}},
```

Comment:
can you explain this?

Reply:
@pankaj-ag Cron to update daily usage before the end of the day